### DISCUSSION TITLE
Scrawler only downloads a fraction of what is previously downloaded only just before

### DISCUSSION THREAD

**Discussion by OneSteelTank** [AAndyProgram/SCrawler#89] [Q&A]
I initially used SC to download an account. It downloaded about 601 files. I wanted to test something unrelated so I copied the files to a different location and then deleted the original folders through the GUI. But now when I download the same profile it only downloads 65 folders. Any ideas?

**Comment by OneSteelTank** [comment]
I realized I messed up my redgif token. Replaced it but it only downloads 131

**Reply by AAndyProgram** [reply]
Is any data in the log?

>and then deleted the original folders through the GUI

Idk. If you delete a user via SCrawler, all content, including data files, must be deleted. Perhaps something happened during the deletion and the data files were not deleted. This is only way why you cannot download full profile again. Check user folder - settings. If it contains `User_UserName_Data.xml` and/or `User_UserName_Posts.txt` files, this is the answer why you can't download.
